tv/components/RoomDisplay.tsx

```diff
@@ -76,7 +92,15 @@ const RoomDisplay = ({ room, currentSession, nextSession }: RoomProps): JSX.Elem
                 </div>
               </div>
               <div className="right-content">
-                <Image className="checkin-code" src={qr} alt="check-in" width={160} height={160} />
+                <Image
+                  className="next-session-qr-code"
```

Comment:
Since both QR codes have exactly the same CSS styles applied to them, it is not necessary to have 2 CSS classes. One would be enough and it could be named "qr-code".